terraform/context_apply_test.go

```diff
@@ -8980,12 +8980,6 @@ Outputs:
 
 result_1 = hello
 result_3 = hello world
-
-module.child:
```

Comment:
Why did this get removed from this test?

Reply:
These only exist temporarily during evaluation, and are always stripped during serialization. This style of test fixture is using the in-memory state, and is adapted to work with the old pre-0.12 format, which unfortunately still has some of the old quirks like showing the module outputs.